app/assets/javascripts/proxy_status/logs.js

```diff
@@ -0,0 +1,97 @@
+function filterLogsReset() {
+  var table = $('#table-proxy-status-logs').dataTable();
+  for (var i = 0; i <= 2; i++) {
+    table.fnFilter('', i);
```

Comment:
You can just use `$('#table-proxy-status-logs').dataTable().fnFilter('');` instead of looping over the columns to clear all the filters.